CONTRIBUTING.md

```diff
@@ -13,14 +13,98 @@ Check out the [Projects](https://github.com/jMonkeyEngine/jmonkeyengine/projects
 
 p.s. We will try hold ourselves to a [certain standard](http://www.defmacro.org/2013/04/03/issue-etiquette.html) when it comes to GitHub etiquette. If at any point we fail to uphold this standard, let us know.
 
-#### Core Contributors
+There are many ways
+to submit a pull request (PR) to the "jmonkeyengine" project repository,
+depending on your knowledge of Git and which tools you prefer.
+
+<details>
+    <summary>
+        <b>Click to view step-by-step instructions for a reusable setup
+        using a web browser and a command-line tool such as Bash.</b>
+    </summary>
+
+The setup described here allows you to reuse the same local repo for many PRs.
+
+#### Prerequisites
+
+These steps need only be done once...
+
+1. Create a personal account at GitHub, if you don't already have one:
+  + Browse to https://github.com/
+  + Click on the "Sign up" button in the upper right.
+  + Follow the instructions.
+2. Sign into your GitHub account, if you're not already signed in:
+  + Browse to https://github.com/
+  + Click on the "Sign in" button in the upper right.
+  + Follow the instructions.
+3. Create a GitHub access token, if you don't already have one:
+  + Browse to https://github.com/settings/tokens
+  + Click on the "Generate new token" button in the upper right.
+  + Follow the instructions.
+  + When specifying the scope of the token, check the box labeled "repo".
+  + Copy the generated token to a secure location from which you can
+    easily paste it into your command-line tool.
+4. Create your personal fork of the "jmonkeyengine" repository at GitHub,
+   if you don't already have one:
+  + Browse to https://github.com/jMonkeyEngine/jmonkeyengine
+  + Click on the "Fork" button (upper right)
+  + Follow the instructions.
+  + If offered a choice of locations, choose your personal account.
+5. Clone the fork to your development system:
+  + `git clone https://github.com/` ***yourGitHubUserName*** `/jmonkeyengine.git`
+  + As of 2021, this step consumes about 1.3 GBytes of filesystem storage.
+6. Create a local branch for tracking the project repository:
+  + `cd jmonkeyengine`
+  + `git remote add project https://github.com/jMonkeyEngine/jmonkeyengine.git`
+  + `git fetch project`
+  + `git checkout -b project-master project/master`
+
+#### PR process
+
+1. Create an up-to-date local branch for your PR changes:
+  + `git checkout project-master`
+  + `git pull`
+  + `git checkout -b myPrBranch` (replace "myPrBranch" with a descriptive name)
+2. Make your changes in the working tree.
+3. Test your changes.
+   Testing should, at a minimum, include building the Engine from scratch:
+  + `./gradlew clean build`
+4. Add and commit your changes to your local PR branch.
+5. Push the PR commits to your fork at GitHub:
+  + `git push --set-upstream origin ` ***yourPrBranchName***
```

Comment:
I might quibble with the use of PR here to describe the branch. This is a "project branch." PR refers specifically the process of asking to merge such a branch into the master/main branch. It's entirely possible that I'm being too pendantic here, though.

Reply:
Good point. I'll rephrase this.